pipelines/README.md

```diff
@@ -0,0 +1,15 @@
+# Pipeline Definitions
+
+This directory contains pipeline definition YAML files which are used when processing data with the LSST Science Pipelines.
+
+The pipelines defined here come in three flavors: camera-specific (within named directories), camera-agnostic (top-level, if any), and building-block ingredients (within the [\_ingredients](_ingredients) directory).
+Pipelines within the ingredients directory tend to be imported by other pipelines, and are not intended to be used directly by end-users for science purposes.
+
+The `pipetask build` command can be used to expand a pipeline YAML and resolve any imports for the purposes of visualizing it.
+For example, to visualize the `processCcd` subset from the [LSSTCam-imSim ApVerify pipeline](https://github.com/lsst/ap_verify/blob/main/pipelines/LSSTCam-imSim/ApVerify.yaml) pipeline, run:
```

Comment:
ImSim is the colloquial name of the instrument:
```suggestion
For example, to visualize the `processCcd` subset from the [ImSim ApVerify pipeline](https://github.com/lsst/ap_verify/blob/main/pipelines/LSSTCam-imSim/ApVerify.yaml) pipeline, run:
```

Reply:
As on the other PR, I think it may be nicer to label this `LSSTCam-imSim` in line with [DM-36796](https://jira.lsstcorp.org/browse/DM-36796). Either way, we should still refer to `LSSTCam-imSim` at some point to help discoverability of these pipeline definitions outside of clicking this link.